frontend/integration-tests/tests/kubevirt/vm.wizard.scenario.ts

```diff
@@ -0,0 +1,117 @@
+/* eslint-disable no-undef */
+
+import { execSync } from 'child_process';
+import { browser, by, ExpectedConditions as until } from 'protractor';
+import { appHost, testName } from '../../protractor.conf';
+import { resourceRowsPresent, filterForName, deleteRow, isLoaded, createItemButton } from '../../views/crud.view';
+import * as vmView from '../../views/kubevirt/vm.view';
+
+
+describe('Kubevirt create VM using wizard', () => {
+  const leakedResources = new Set<string>();
+  const provisionSource = 'PXE';
+  const operatingSystem = 'fedora28';
+  const flavor = 'small';
+  const workloadProfile = 'generic';
+  const networkDefinitionName = `${testName}-ovs-net-1`;
+  const pxeInterface = 'eth1';
+  const testNAD = {
+    apiVersion: 'k8s.cni.cncf.io/v1',
+    kind: 'NetworkAttachmentDefinition',
+    metadata: {
+      name: networkDefinitionName,
+      namespace: testName,
+    },
+    spec: {
+      config: '{ "cniVersion": "0.3.1", "type": "ovs", "bridge": "br0" }',
+    },
+  };
+
+  beforeAll(async() => {
+    execSync(`echo '${JSON.stringify(testNAD)}' | kubectl create -f -`);
+  });
+
+  afterAll(async() => {
+    execSync(`kubectl delete -n ${testName} net-attach-def ${networkDefinitionName}`);
+    const leakedArray: Array<string> = [...leakedResources];
+    if (leakedArray.length > 0) {
+      console.error(`Leaked ${leakedArray.join()}`);
+      leakedArray.map(r => JSON.parse(r) as {name: string, namespace: string, kind: string})
+        .forEach(({name, namespace, kind}) => {
+          try {
+            execSync(`kubectl delete -n ${namespace} --cascade ${kind} ${name}`);
+          } catch (error) {
+            console.error(`Failed to delete ${kind} ${name}:\n${error}`);
+          }
+        });
+    }
+  });
+
+  it('Navigates to VMs', async() => {
+    await browser.get(`${appHost}/k8s/all-namespaces/virtualmachines`);
+    await isLoaded();
+  });
+
+  it('Opens VM wizard', async() => {
+    await createItemButton.click().then(() => vmView.createWithWizardLink.click());
+  });
+
+  it('Configures VM Basic Settings', async() => {
+
+    await browser.wait(until.presenceOf(vmView.nameInput), 10000);
+    await vmView.nameInput.sendKeys(testName);
```

Comment:
having a different vm name from namespace would be better IMO (`-vm` suffix?)

Reply:
Good point, I added 'vm-' prefix as I think it looks a bit better than a suffix :).